pkgs/servers/web-apps/pump.io/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, makeWrapper, callPackage, nodejs, python, utillinux, graphicsmagick }:
+
+with stdenv.lib;
+
+let
+  nodePackages = callPackage (import ../../../top-level/node-packages.nix) {
+    inherit stdenv nodejs fetchurl fetchgit;
+    neededNatives = [ python ] ++ optional stdenv.isLinux utillinux;
+    self = nodePackages;
+    generated = ./package.nix;
```

Comment:
could you rename ``package.nix`` to ``node-packages.nix`` just so it is obvious that this are node packages, like in top-level folder/